doc/release-notes.md

```diff
@@ -105,6 +105,15 @@ RPC
 Tests
 -----
 
+Files
+-----
+
+- The list of banned hosts and networks (via `setban` RPC) is now saved on disk
+  in JSON format in `banlist.json` instead of `banlist.dat`. Both files are
+  read on startup and contents merged. Updateds are only written to the new
+  `banlist.json`. A future version of Bitcoin Core may completely ignore
+  `banlist.dat`. (#20966)
```

Comment:
Please add the release notes to https://github.com/bitcoin-core/bitcoin-devwiki/wiki/%2320852-Release-notes-snippet

Otherwise they will have to be removed here after merge, then added to the `0.21` branch. (Release notes are only added to the point releases, not the next major version that also includes those fixes)

Reply:
I guess after/if this PR is merged?